src/Core/Framework/Adapter/Cache/CacheTracer.php

```diff
@@ -1,44 +0,0 @@
-<?php declare(strict_types=1);
-
-namespace Shopware\Core\Framework\Adapter\Cache;
-
-use Shopware\Core\Framework\Adapter\Translation\AbstractTranslator;
-use Shopware\Core\Framework\Log\Package;
-use Shopware\Core\Framework\Plugin\Exception\DecorationPatternException;
-use Shopware\Core\System\SystemConfig\SystemConfigService;
-
-/**
- * @extends AbstractCacheTracer<mixed|null>
- */
-#[Package('framework')]
-class CacheTracer extends AbstractCacheTracer
```

Comment:
this was not deprecated, I would backport that for 6.6, as the parent abstract class was deprecated